rotkehlchen/exchanges/constants.py

```diff
@@ -16,12 +16,11 @@
     Location.ICONOMI,
     Location.KRAKEN,
     Location.KUCOIN,
-    Location.FTX,
-    Location.FTXUS,
     Location.INDEPENDENTRESERVE,
     Location.POLONIEX,
     Location.OKX,
 ]
 EXCHANGES_WITH_PASSPHRASE = (Location.COINBASEPRO, Location.KUCOIN, Location.OKX)
+DEAD_EXCHANGES = [Location.FTX, Location.FTXUS]
```

Comment:
Since this is constant is better to use tuples than lists. Reasons: performance and avoid undesired mutability

Reply:
+ 1. Do this please @nebolax